pyproject.toml

```diff
@@ -54,10 +54,15 @@ fortran = [
     # the Fortran parser from the (unfinished) third-edition
     # f2py, as linked below. This package is not on the package index, AFAIK.
     # -AK, 2024-08-02
-    "f2py @ git+https://github.com/pearu/f2py.git",
+    # Fork for https://github.com/pearu/f2py/pull/63
+    "f2py @ git+https://github.com/inducer/f2py.git@pyproject",
     "ply>=3.6",
 ]
+
+[dependency-groups]
```

Comment:
What's the benefit for `dependency-groups`? I haven't looked into it much :(

Reply:
Defined in https://peps.python.org/pep-0735/. The PEP describes the design decisions. It seems that dependency groups are the "hot new" way to do dev dependencies. A markedly useful aspect here is that, unlike the usual "extras", they permit *only* the dev dependencies to be installed, without fully installing loopy.

Reply:
> they permit only the dev dependencies to be installed

I see why that's quite useful here, thanks :D